src/main/java/com/Crowdsourcing/loot/CrowdsourcingLoot.java

```diff
@@ -0,0 +1,208 @@
+package com.Crowdsourcing.loot;
+
+import com.Crowdsourcing.CrowdsourcingManager;
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.ChatMessageType;
+import net.runelite.api.Client;
+import net.runelite.api.InventoryID;
+import net.runelite.api.ItemContainer;
+import net.runelite.api.ItemID;
+import net.runelite.api.Skill;
+import net.runelite.api.events.ChatMessage;
+import net.runelite.client.callback.ClientThread;
+import net.runelite.client.eventbus.Subscribe;
+import net.runelite.client.game.ItemStack;
+import net.runelite.client.plugins.loottracker.LootReceived;
+import net.runelite.http.api.loottracker.LootRecordType;
+
+@Slf4j
+public class CrowdsourcingLoot {
+	@Inject
+	Client client;
+
+	@Inject
+	ClientThread clientThread;
+
+	@Inject
+	CrowdsourcingManager manager;
+
+	private static final Map<Integer, String> VARBITS_CA = new HashMap<>() {
+		{
+			put(12863, "easyCA");
+			put(12864, "mediumCA");
+			put(12865, "hardCA");
+			put(12866, "eliteCA");
+			put(12867, "masterCA");
+			put(12868, "grandmasterCA");
+		}
+	};
+	private static final int CA_CLAIMED = 2;
+
+	private static final Map<Integer, String> VARBITS_CLUE_WARNINGS = new HashMap<>() {
+		{
+			put(10693, "beginnerClueWarn");
+			put(10694, "easyClueWarn");
+			put(10695, "mediumClueWarn");
+			put(10723, "hardClueWarn");
+			put(10724, "eliteClueWarn");
+			put(10725, "masterClueWarn");
+		}
+	};
+	private static final int CLUE_WARNING_ENABLED = 0;
+
+	// Hunters' loot sacks
+	private static final String HUNTERS_LOOT_SACK_BASIC = "Hunters' loot sack (basic)";
+	private static final String HUNTERS_LOOT_SACK_ADEPT = "Hunters' loot sack (adept)";
+	private static final String HUNTERS_LOOT_SACK_EXPERT = "Hunters' loot sack (expert)";
+	private static final String HUNTERS_LOOT_SACK_MASTER = "Hunters' loot sack (master)";
+
+	// Clues
+	private static final Pattern CLUE_MESSAGE = Pattern.compile("You have a sneaking suspicion.*");
```

Comment:
I checked that this will only match clue messages as of now based on our existing messages data 👍